app/views/profiles/_profile.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="medium-12 columns">
+  <h3>Profile</h3>
+    <form data-abide>
+      <div class="row">
+        <div class="medium-12 columns username-field">
+          <label>First Name
+            <input type="text" required />
+          </label>
+          <small class="error">Required</small>
+        </div>
+      </div>
+      <div class="row">
+        <div class="medium-12 columns username-field">
```

Comment:
This form came directly from the styleguide and will need to be changed to a form_for (unless there are other suggestions) when we hook up the form. I put this form from the styleguide in because an un-hooked form_for currently breaks the site. I should have called this out in my initial PR comment.